src/main/kotlin/org/partiql/scribe/targets/redshift/RedshiftAstToSql.kt

```diff
@@ -123,4 +131,52 @@ public open class RedshiftAstToSql(context: ScribeContext) : AstToSql(context) {
             else -> super.visitDataType(node, tail)
         }
     }
+
+    /**
+     * The PartiQL plan and thus AST will fully-qualify ORDER BY variable references and paths with an implicit
+     * binding tuple name for set operations. For Redshift, there is no implicit binding tuple name for the set ops. So
+     * we must remove the prefix binding tuple name from paths.
+     *
+     * For example, the query
+     *   (SELECT a FROM ...) UNION (SELECT a FROM ...) ORDER BY a
+     * will have an extra qualification
+     *   (SELECT a FROM ...) UNION (SELECT a FROM ...) ORDER BY "_1".a
+     * The additional qualification is invalid Redshift, hence why we remove it.
+     *
+     * Additionally, Redshift does not support path expressions in ORDER BY applied to set operations. See
+     * #[order-by-10] for an example of such an unsupported query.
```

Comment:
self-review: this is one edge case that Redshift does not support. From the [tests](https://github.com/partiql/partiql-scribe/blob/order-by/src/test/resources/inputs/basics/order-by.sql#L31-L32):
```
--#[order-by-10]
(SELECT T.flds FROM EXCLUDE_T AS T) UNION (SELECT T.flds FROM EXCLUDE_T AS T) ORDER BY flds.c.field_x;
```

 When the `ORDER BY` for a set operation includes an expression that paths into a nested struct, Redshift gives the following error:

```
ERROR: ORDER BY on a UNION/INTERSECT/EXCEPT result must be on one of the result columns
```

This transpiled query works for Trino and Spark though.